avalanche/training/templates/base.py

```diff
@@ -154,6 +158,10 @@ def eval(
 
         self._after_eval(**kwargs)
 
+        # reset for faster serialization
+        self.current_eval_stream = None
```

Comment:
I think it's better if we put everything inside a cleanup method called from _after_train and _after_eval